From 707ac921eff5d8f7ec9353ec162c2461bc4b6b9b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
Date: Sun, 28 Aug 2022 19:52:57 +0200
Subject: [PATCH 1/2] Fix invalid application name

error: 'io.github.xaos-project.XaoS' is not a valid application name:
Only last name segment can contain -
---                                         | 2 +-
 ...os-project.desktop => io.github.xaos_project.XaoS.desktop} | 0
 xdg/xaos.appdata.xml                                          | 4 ++--
 3 files changed, 3 insertions(+), 3 deletions(-)
 rename xdg/{io.github.xaos-project.desktop => io.github.xaos_project.XaoS.desktop} (100%)

diff --git a/xdg/io.github.xaos-project.desktop b/xdg/io.github.xaos_project.XaoS.desktop
similarity index 100%
rename from xdg/io.github.xaos-project.desktop
rename to xdg/io.github.xaos_project.XaoS.desktop
diff --git a/xdg/xaos.appdata.xml b/xdg/xaos.appdata.xml
index b2f40195..0dd064d0 100644
--- a/xdg/xaos.appdata.xml
+++ b/xdg/xaos.appdata.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
-  <id>io.github.xaos-project</id>
+  <id>io.github.xaos_project.XaoS</id>
   <metadata_license>FSFAP</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>XaoS</name>
@@ -21,7 +21,6 @@
     </p>
   </description>
 
-  <launchable type="desktop-id">io.github.xaos-project.desktop</launchable>
 
   <screenshots>
     <screenshot type="default">
@@ -48,6 +47,8 @@
     <binary>xaos</binary>
   </provides>
 
+  <launchable type="desktop-id">io.github.xaos_project.XaoS.desktop</launchable>
+
   <releases>
     <release version="4.2.1" date="2021-02-01">
       <description>

From 0727c670fe74434859d891fe8eba51381e659623 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
Date: Tue, 30 Aug 2022 11:18:40 +0200
Subject: [PATCH 2/2] Add Open Age Rating

---
 xdg/xaos.appdata.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xdg/xaos.appdata.xml b/xdg/xaos.appdata.xml
index 0dd064d0..d594faf0 100644
--- a/xdg/xaos.appdata.xml
+++ b/xdg/xaos.appdata.xml
@@ -21,6 +21,7 @@
     </p>
   </description>
 
+  <content_rating type="oars-1.1" />
 
   <screenshots>
     <screenshot type="default">
